### PR TITLE
Fix unhandled exception for authn login MODLOGIN-157

### PR DESCRIPTION
**Pull request by adamdickmeiss**
https://issues.folio.org/browse/MODLOGIN-157

Get rid of runOnContext.. They are redundant here and then exceptions are thrown in them, there's might be no response.

Use Okapi definitions for headers.

### CODE REVIEW

File: src/test/java/org/folio/logintest/LogEventsApiTest.java [hunk 1]
```diff
@@ -113,6 +111,14 @@ public void setUp(TestContext context) {
     });
   }
 
+  @Test
+  public void testMissingOkapiUrl() {
+    JsonObject logEven = getLogEven("tenant", UUID.randomUUID().toString(), LogEvent.EventType.PASSWORD_RESET.toString());
```
Comment (julianladisch) [inline src/test/java/org/folio/logintest/LogEventsApiTest.java:116]: This exists before but can you rename `getLogEven` to `getLogEvent`?

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

File: src/test/java/org/folio/logintest/LogEventsApiTest.java (hunk 1 above)
Comment (adamdickmeiss) [reply src/test/java/org/folio/logintest/LogEventsApiTest.java:116]: Sure

Comment (adamdickmeiss) [reply src/test/java/org/folio/logintest/LogEventsApiTest.java:116]: Sure

**Pull request merged**